### PR TITLE
Handle the case where email is used

### LINKED ISSUE #140: Check email duplicate in registration
Currently, two account can have the same email. This shouldn't be allowed, because one person can only have one account.

Though a person can have more than one email, this is a way to mitigate the problem.

### PR DESCRIPTION
**Pull request by Huy-Ngo**
Resolve #140

### CODE REVIEW

**Comment by Huy-Ngo** [comment]
OK this one hopefully pass.

**Comment by Huy-Ngo** [comment]
Yay it passed

File: src/acanban/auth.py [hunk 1]
```diff
@@ -80,6 +80,10 @@ async def add_user(username: str, password: str,
     user = {'username': username, 'password': crypt(password),
             'name': name, 'email': email, 'role': role}
     async with current_app.db_pool.connection() as connection:
+        user_with_mail = await r.table('users').filter(
+            {'email': user['email']}).count().run(connection)
```
Comment (McSinyx) [inline src/acanban/auth.py:84]:
```suggestion
            {'email': email}).count().run(connection)
```

**Review by McSinyx — APPROVED** [review]
The overall change looks good enough.

**Pull request merged**